buildstockbatch/gcp/gcp.py

```diff
@@ -884,7 +893,86 @@ def start_combine_results_job_on_cloud(self, results_dir, do_timeseries=True):
                     f"{self.postprocessing_job_console_url}",
                     exc_info=True,
                 )
-                break
+                return
+
+        # Monitor job/execution status, starting by polling the Job for an Execution
+        logger.info("Waiting for execution to begin...")
+        job_start_time = datetime.now()
+        jobs_client = run_v2.JobsClient()
+        job = jobs_client.get_job(name=self.postprocessing_job_name)
+        while not job.latest_created_execution:
```

Comment:
Could you get the execution from the return value of `run_job` above instead?

Reply:
`run_job` (and `create_job`) [return an Operation](https://cloud.google.com/python/docs/reference/run/latest/google.cloud.run_v2.services.jobs.JobsClient#google_cloud_run_v2_services_jobs_JobsClient_run_job), not a Job (which might have an execution in it), from what I can tell.